README.md

```diff
@@ -1 +1,27 @@
-# Microverse-Project-1st-week-
\ No newline at end of file
+Project Name: Microverse Project (1st Week)
+Description the project. (Microverse Project task for 1st week)
+
+Major languages: HTML & CSS
+
+Prerequisites
+Setup
+Install: VS Code, Chorme Browser
+Authors
+👤 Author1
+
+GitHub: @umair-rao
+LinkedIn: https://www.linkedin.com/in/engr-umair-muqeem/
+
```

Comment:
Great Updating your README file 👀 , But kindly follow the [Template](https://github.com/microverseinc/readme-template), so it looks more professional.🤓

Reply:
File updated according to template.